app/boards/shields/two_percent_milk/two_percent_milk.overlay

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/ {
+	chosen {
+		zmk,kscan = &kscan0;
+	};
+
+	kscan0: kscan {
+		compatible = "zmk,kscan-gpio-direct";
+
+		label = "KSCAN";
+
+		input-gpios
+            = <&pro_micro 4 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>
+            , <&pro_micro 5 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>
```

Comment:
```suggestion
			= <&pro_micro 4 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>
			, <&pro_micro 5 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>
```

Reply:
Format change committed and squashed in [96b5d07](https://github.com/zmkfirmware/zmk/commit/96b5d075843ac5447ceeedffdf40c2cce8372ddd).